x-pack/plugins/data_visualizer/public/plugin.ts

```diff
@@ -37,6 +38,7 @@ export interface DataVisualizerStartDependencies {
   security?: SecurityPluginSetup;
   share: SharePluginStart;
   lens?: LensPublicStart;
+  charts?: ChartsPluginStart;
```

Comment:
charts is now a required plugin and so doesn't need to be optional here.

Reply:
Changed in https://github.com/elastic/kibana/pull/121286/commits/69c0bd38a8bbba9347ffc607f2fa6ad3fa55ac0d